tests/test_extend_schema.py

```diff
@@ -271,3 +291,80 @@ def raw_action(self, request):
     assert extended_action_op['$ref'].endswith('C')
     assert list_op['items']['$ref'].endswith('B')
     assert raw_action_op['$ref'].endswith('B')
+
+
+def test_extend_schema_on_view_and_method_with_examples(no_warnings):
```

Comment:
since this is a very big test and the other tests are already large, we should put it in its own test `tests/test_examples.py`.

that way we spare the existing tests from large changes and have a dedicated test and yaml for comparison. also a reference yaml is easier to inspect than 10 asserts.